<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="inicio_tenant-20231210170512" tests="1" file=".py" time="21.673" timestamp="2023-12-10T17:05:34" failures="0" errors="1" skipped="0">
	<testcase classname="inicio_tenant" name="test_login_user_1" time="21.673" timestamp="2023-12-10T17:05:34" file="inicio.py" line="32">
		<error type="NoSuchWindowException" message="Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=120.0.6099.71)
Stacktrace:
	GetHandleVerifier [0x00007FF729E74D02+56194]
	(No symbol) [0x00007FF729DE04B2]
	(No symbol) [0x00007FF729C876AA]
	(No symbol) [0x00007FF729C60AFD]
	(No symbol) [0x00007FF729CFC9AB]
	(No symbol) [0x00007FF729D1201F]
	(No symbol) [0x00007FF729CF5C23]
	(No symbol) [0x00007FF729CC4A45]
	(No symbol) [0x00007FF729CC5AD4]
	GetHandleVerifier [0x00007FF72A1ED5BB+3695675]
	GetHandleVerifier [0x00007FF72A246197+4059159]
	GetHandleVerifier [0x00007FF72A23DF63+4025827]
	GetHandleVerifier [0x00007FF729F0F029+687785]
	(No symbol) [0x00007FF729DEB508]
	(No symbol) [0x00007FF729DE7564]
	(No symbol) [0x00007FF729DE76E9]
	(No symbol) [0x00007FF729DD8094]
	BaseThreadInitThunk [0x00007FFB1102257D+29]
	RtlUserThreadStart [0x00007FFB1232AA58+40]
"><![CDATA[Traceback (most recent call last):
  File "c:\Users\luist\repos\Automatizaciones-SILOHUB-\inicio.py", line 45, in test_login_user_1
    insertButton = driver.find_element(By.XPATH,"/html/body/app-root/app-login-main/div/div[2]/div/app-login-form/div/div/div[1]/div/div[2]/form/div[4]/app-button/button")
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=120.0.6099.71)
Stacktrace:
	GetHandleVerifier [0x00007FF729E74D02+56194]
	(No symbol) [0x00007FF729DE04B2]
	(No symbol) [0x00007FF729C876AA]
	(No symbol) [0x00007FF729C60AFD]
	(No symbol) [0x00007FF729CFC9AB]
	(No symbol) [0x00007FF729D1201F]
	(No symbol) [0x00007FF729CF5C23]
	(No symbol) [0x00007FF729CC4A45]
	(No symbol) [0x00007FF729CC5AD4]
	GetHandleVerifier [0x00007FF72A1ED5BB+3695675]
	GetHandleVerifier [0x00007FF72A246197+4059159]
	GetHandleVerifier [0x00007FF72A23DF63+4025827]
	GetHandleVerifier [0x00007FF729F0F029+687785]
	(No symbol) [0x00007FF729DEB508]
	(No symbol) [0x00007FF729DE7564]
	(No symbol) [0x00007FF729DE76E9]
	(No symbol) [0x00007FF729DD8094]
	BaseThreadInitThunk [0x00007FFB1102257D+29]
	RtlUserThreadStart [0x00007FFB1232AA58+40]

]]></error>
	</testcase>
</testsuite>
